dev/WindowsAppRuntime_MSIXInstallFromPath/main.cpp

```diff
@@ -138,6 +140,12 @@ HRESULT JustDoIt(PCWSTR path) noexcept try
     auto inventory{ fpath / L"MSIX.inventory" };
     std::string lineUtf8;
     std::ifstream f{ inventory };
+
+    if (!std::filesystem::exists(inventory))
+    {
+        THROW_HR_MSG(HRESULT_FROM_WIN32(ERROR_FILE_NOT_FOUND), "inventory file doesn't exist: %s", inventory);
```

Comment:
1. Message text here is redundant with the FILE_NOT_FOUND error code
2. WIL *_MSG(....%s...) == narrow string but `inventory` is a wide string
3. inventory is a `std::filesystem::path` but %s expects C string

`THROW_HR_MSG(HRESULT_FROM_WIN32(ERROR_FILE_NOT_FOUND), "%ls", inventory.c_str());`